### DISCUSSION TITLE
Loading csv with date format

### DISCUSSION THREAD

**Discussion by yanivpaz** [grafana/grafana-csv-datasource#125] [Q&A]
Any idea if CSV files like below can be loaded using the local method?

"2021-09-02 15:04",1
"2021-09-01 15:04",24

I am getting the message  "Data does not have a time field" 
Thanks

**Comment by VladimirBessonov** [comment]
any answer you received? 
I am having the same issue? 
data and time is not recognized as "time"
strange enough  - numbers are also not recognized as numbers. when I switch ignore tap, nothing is displayed.

**Comment by marcusolsson** [comment]
All fields are parsed as strings by default. You need to define a schema in order to set the type. Since the CSV is parsed on the server, if the timestamps don't have time zone information, UTC is assumed. Local time is unfortunately not supported as of now.

**Reply by Kikkomanq** [reply]
Hi, 
Can you describe how to -define a schema in order to set the type, since I have same issue with "Data does not have a time field"?
Here's the csv, based on previous post I went for format yyyy-mm-dd hh:mm:ss:
![image](https://user-images.githubusercontent.com/16452895/143022778-71b304f7-1f4a-4db8-a676-dd8d1dbdd72e.png)

**Reply by niccoxx** [reply]
Hi,
I've the same question. In a csv file, how I should write a date field. Now I try with many configuration but I've always a blank field.

**Reply by a-rodion** [reply]
Define time-field as String on `Fields` tab and then transform it to Time on `Transform` tab.

**Comment by a-rodion** [comment]
@marcusolsson 
>Local time is unfortunately not supported as of now.

Really need converting to local time for properly working of filter.

**Reply by marcusolsson** [reply]
Hey! 👋  I'm not the maintainer of this plugin anymore (I've donated the plugin to the Grafana team). If you create an issue, I'm sure the Grafana team will respond.

**Reply by a-rodion** [reply]
Thanks for response!
